index.js

```diff
@@ -0,0 +1,463 @@
+      id: 5,
+      title: 'Professional Art Printing Data',
+      desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      tecnologies: {
+        tech1: 'html',
+        tech2: 'bootstrap',
+        tech3: 'Ruby',
+      },
+      live: 'https://maryam0007.github.io/',
+      source: 'https://maryam0007.github.io/',
+      img: './img/about4-res.png',
+    },
+    {
+      id: 6,
+      title: 'Professional Art Printing Data',
+      desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      tecnologies: {
+        tech1: 'html',
+        tech2: 'bootstrap',
+        tech3: 'Ruby',
+      },
+      live: 'https://maryam0007.github.io/',
+      source: 'https://maryam0007.github.io/',
+      img: './img/about4-res.png',
+    },
+    {
+      id: 7,
+      title: 'Professional Art Printing Data',
+      desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      tecnologies: {
+        tech1: 'html',
+        tech2: 'bootstrap',
+        tech3: 'Ruby',
+      },
+      live: 'https://maryam0007.github.io/',
+      source: 'https://maryam0007.github.io/',
+      img: './img/about4-res.png',
+    },
+  ];
+  // Create pop up section for mob
+  const workdetailsSection = (content) => `
+<div class="pop-up-header" id="popupclosemobile">
+        <span class="mobileclose">&times;</span>
+        <h2>${cardDetails[content].title}</h2>
+      </div>
+      <div>
+        <ul class="pop-up-tags">
+          <li>${cardDetails[content].tecnologies.tech1}</li>
+          <li>${cardDetails[content].tecnologies.tech2}</li>
+          <li>${cardDetails[content].tecnologies.tech3}</li>
+        </ul>
+      </div>
+      <div class="mobile-popup-img">
+      <img src="${cardDetails[content].img}" alt="./img/popup-mob.png">
+      </div>
+      <div class="popup-description">
+        <p>${cardDetails[content].desc}</p>
+      </div>
+      <div class="popup-buttons">
+      <a href="${cardDetails[content].live}" id="see-live"><img src="./img/live-mob.png"></a>
+      <a href="${cardDetails[content].source}" id="see-source"> <img src="./img/source-mob.png"></a>  
+      </div>
+`;
+
+  const details = document.querySelectorAll('#project');
+  // Create pop up section for mobile
+  for (let i = 0; i < details.length; i += 1) {
+    details[i].addEventListener('click', () => {
+      const div = document.createElement('div');
+      div.classList.add = 'pop-up-mobile';
+      div.setAttribute('id', 'popupmobile');
+      div.innerHTML = workdetailsSection(i);
+      const container = document.querySelector('.popmobdiv');
+      container.appendChild(div);
+      const closedetails = document.querySelector('.mobileclose');
+      closedetails.addEventListener('click', () => {
+        container.removeChild(div);
+      });
+    });
+  }
```

Comment:
- Kindly, update your modal style like adding blur on the background, stopping background scroll to stick to the Figma design and have a better UI 👍
### Figma Design 👇
![2023-04-12_17-08](https://user-images.githubusercontent.com/52010789/231485544-d2abc9f1-2350-47ef-87e4-1ad777b65159.png)
### Your Design 👇
![2023-04-12_17-10](https://user-images.githubusercontent.com/52010789/231485584-0be33fa0-39d6-4e03-8594-b138a492f1db.png)